src/frontend/screens/Observation/index.tsx

```diff
@@ -93,17 +98,25 @@ export const ObservationScreen: NativeNavigationComponent<'Observation'> = ({
               <Text style={styles.textNotes}>{observation.tags.notes}</Text>
             </View>
           ) : null}
-          {attachmentUrls.length > 0 && (
+          {(photoAttachmentsUrls.length > 0 || recordings.length > 0) && (
             <MediaScrollView
-              photos={attachmentUrls.map(attachmentData => {
+              audioRecordings={audioUrls.map(attachmentData => {
+                return !attachmentData
+                  ? undefined
+                  : {
+                      uri: attachmentData.url,
+                      createdAt: 0,
+                      duration: 0,
```

Comment:
We're unable to save metadata with audio recording, hence mock values for now. Possibly related to #378